source/docs/software/wpilib-tools/robot-simulation/introduction.rst

```diff
@@ -0,0 +1,45 @@
+Introduction to Robot Simulation
+================================
+
+Often a team may want to test their code without having an actual robot available. WPILib provides teams with the ability to simulate the robot's Hardware Abstraction Layer (HAL) using simple gradle commands.
+
+Enabling Desktop Support
+------------------------
+
+Before you can start robot simulation, it's required to have desktop support enabled. Desktop support can be enabled via checking the  "Enable Desktop Support Checbox" when creating your robot project, or via running "WPILib: Change Desktop Support Enabled Setting" from the command palette. If none of the already listed options work, please `report a bug <https://github.com/wpilibsuite/allwpilib/issues>`__.
```

Comment:
```suggestion
Use of the Desktop Simulator requires Desktop Support to be enabled. This can be done by checking the "Enable Desktop Support Checkbox" when creating your robot project or by running "WPILib: Change Desktop Support Enabled Setting" from the Visual Studio Code command palette. If none of the already listed options work, please `report a bug <https://github.com/wpilibsuite/allwpilib/issues>`__.
```